pom.xml

```diff
@@ -110,6 +110,12 @@
       <artifactId>joda-time</artifactId>
       <version>2.10.3</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>mod-source-record-storage-client</artifactId>
+      <version>2.6.0-SNAPSHOT</version>
```

Comment:
This will need changing prior to the next formal release, as SNAPSHOT dependencies are not intended to be used.